webapp/src/main/webapp/src/app/shared/base.service.ts

```diff
@@ -25,4 +25,8 @@ export abstract class BaseService<T extends BaseEntity> {
 		return this.httpClient.delete<T>(environment.apiBaseUrl + resource + '/' + entity.id);
 	}
 
```

Comment:
This second argument on this  searchCountAndItems should not be of type any, please don't use any, use a concrete type

Reply:
Did you see this comment?